code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -88,7 +88,7 @@
 			return GLOB.xeno_types_tier_four + /datum/xeno_caste/hivemind
 		if(XENO_TIER_FOUR)
 			if(istype(xeno_caste, /datum/xeno_caste/shrike))
-				return list(/datum/xeno_caste/queen, /datum/xeno_caste/king)
+				return GLOB.xeno_types_tier_three + /datum/xeno_caste/queen + /datum/xeno_caste/king
```

Comment:
uh, why is this snowflaked to specifically t3? Kinda funny that they can go rav but not hunter or whatever

Reply:
Arbitrarily, I guess. Shrike has always struck me as around a t2 in powerlevel, so it evolves to a t3 like a t2 can, and xenos generally evolve to one tier specifically.

also having both tiers up at once would probably be annoying to navigate

Reply:
just evo t3 and regress instantly?

Reply:
yeah and what happens if there are no free t3 slots lol? Like its really very snowflake

Reply:
Then you just don't evo ig. Shrike is already snowflaked to evo to queen and king exclusively
Do you want me to add other tiers to evo or change what tier it evos to?